### PR TITLE
Frontend cards

### PR DESCRIPTION
**Pull request by cat-friend**
completed as much as i could without deck feature completed; need to test, check store state, overall debugging

### CODE REVIEW

File: react-app/src/components/DeleteCardFormModal/DeleteCardForm.js [hunk 1]
```diff
@@ -1,18 +1,63 @@
 import React, { useState, useEffect } from "react";
 import { useDispatch, useSelector } from "react-redux";
+import * as cardActions from "../../store/cards";
 
 function DeleteCardForm({ payload }) {
     const setShowModal = payload;
+    const currCard = useSelector(state => state.cards);
+    const dispatch = useDispatch();
+    const currUserId = useSelector(state => state.session.userId);
```
Comment (DanielLaV) [inline react-app/src/components/DeleteCardFormModal/DeleteCardForm.js:9]: I think this is supposed to be state.session.user.id

File: react-app/src/components/DeleteCardFormModal/DeleteCardForm.js [hunk 2]
```diff
@@ -1,18 +1,63 @@
...
+    const currDeckId = useSelector(state => state.decks.userId);
```
Comment (DanielLaV) [inline react-app/src/components/DeleteCardFormModal/DeleteCardForm.js:10]: I don't think state.decks.userId is a thing

File: react-app/src/components/EditCardFormModal/EditCardForm.js [hunk 3]
```diff
@@ -1,39 +1,57 @@
...
 
   const handleSubmit = (e) => {
     e.preventDefault();
     setErrors([]);
     const payload = {
       front,
       back,
-      deck_id: 1
+      deck_id: 1,
+      cardId: 1
```
Comment (DanielLaV) [inline react-app/src/components/EditCardFormModal/EditCardForm.js:28]: I'm guessing these are set to 1 for testing purposes?

**Review by DanielLaV — CHANGES_REQUESTED** [review]
I added some comments about possible problems. Also, the <Test /> component can be deleted from App.js

File: react-app/src/components/EditCardFormModal/EditCardForm.js (hunk 3 above)
Comment (cat-friend) [reply react-app/src/components/EditCardFormModal/EditCardForm.js:28]: yes! will remove (and update to appropriate, dynamic values) once i can integrate and test elsewhere

File: react-app/src/components/DeleteCardFormModal/DeleteCardForm.js (hunk 1 above)
Comment (cat-friend) [reply react-app/src/components/DeleteCardFormModal/DeleteCardForm.js:9]: yes, you're right

File: react-app/src/components/DeleteCardFormModal/DeleteCardForm.js (hunk 2 above)
Comment (cat-friend) [reply react-app/src/components/DeleteCardFormModal/DeleteCardForm.js:10]: it's not; it should be state.deck(s).[some other thing to key into the appropriate deck].user_id; will mark for change. can fix once i can see what the store looks like on the appropriate page.

**Review by DanielLaV — APPROVED** [review]
B-E-A-utiful. Great stuff

**Pull request merged**